woningwaardering/stelsels/zelfstandige_woonruimten/gemeenschappelijke_parkeerruimten/gemeenschappelijke_parkeerruimten.py

```diff
@@ -0,0 +1,217 @@
+                )
+                continue
+
+            if ruimte.detail_soort.code in [
+                Ruimtedetailsoort.parkeerterrein.code,
+                Ruimtedetailsoort.parkeergarage.code,
+            ]:
+                logger.warning(
+                    f"Ruimte {ruimte.naam} ({ruimte.id}) is een {Ruimtedetailsoort.parkeerterrein.naam if ruimte.detail_soort.code==Ruimtedetailsoort.parkeerterrein.code else Ruimtedetailsoort.parkeergarage.naam} en kan momenteel niet gewaardeerd worden in de woningwaardering package. Voeg een parkeerplek los toe aan de eenheden om deze in aanmerking te laten komen voor een waardering onder {Woningwaarderingstelselgroep.gemeenschappelijke_parkeerruimten.naam}. Raadpleeg docs/implementatietoelichting-beleidsboeken/zelfstandige_woonruimten.md voor meer informatie."
+                )
+                continue
+
+            if ruimte.detail_soort.code not in [
+                Ruimtedetailsoort.parkeervak_auto_binnen.code,
+                Ruimtedetailsoort.carport.code,
+                Ruimtedetailsoort.parkeervak_auto_buiten_niet_overdekt.code,
+            ]:
+                logger.info(
+                    f"Ruimte {ruimte.naam} ({ruimte.id}) is geen gemeenschappelijke parkeerruimte en wordt niet gewaardeerd onder rubriek {Woningwaarderingstelselgroep.gemeenschappelijke_parkeerruimten.naam}."
+                )
+                continue
+
+            if ruimte.oppervlakte is None:
+                warnings.warn(
+                    f"Ruimte {ruimte.naam} ({ruimte.id}) heeft geen 'oppervlakte'."
+                )
+                continue
+
+            if ruimte.gedeeld_met_aantal_eenheden is None:
+                warnings.warn(
+                    f"Ruimte {ruimte.naam} ({ruimte.id}) heeft geen 'gedeeld_met_aantal_eenheden'. Zet 'gedeeld_met_aantal_eenheden' >= 2 wanneer de ruimte gedeeld is. 'gedeeld_met_aantal_eenheden' op 0 of 1 wordt beschouwd als niet gedeeld."
+                )
+                continue
+
+            if not ruimte.oppervlakte >= 12.0:
+                logger.info(
+                    f"Ruimte {ruimte.naam} ({ruimte.id}) voldoet niet aan de eis van 12m2 voor een parkeervak."
+                )
+                continue
+
+            for type_parkeeruimte, punten in self.parkeertype_punten_mapping[
+                ruimte.detail_soort.code
+            ].items():
+                criterium = f"{type_parkeeruimte}"
+
+                if heeft_bouwkundig_element(
+                    ruimte, Bouwkundigelementdetailsoort.laadpaal
+                ):
+                    punten += Decimal("2.0")
+                    criterium += " + laadpaal"
+
+                if ruimte.gedeeld_met_aantal_eenheden >= 2:
+                    criterium += f" (gedeeld met {ruimte.gedeeld_met_aantal_eenheden} {'eenheid' if ruimte.gedeeld_met_aantal_eenheden == 1 else 'eenheden'})"
```

Comment:
```suggestion
                    criterium += f" (gedeeld met {ruimte.gedeeld_met_aantal_eenheden} eenheden)"
```
in de regel erboven filteren we al op `ruimte.gedeeld_met_aantal_eenheden >=2`